### DISCUSSION TITLE
Mapping vim style meta keyboard sequences vs emacs style chords?

### DISCUSSION THREAD

**Discussion by suykerbuyk** [kovidgoyal/kitty#5700] [Ideas]
First I want to congratulate you on almost single-handedly creating the most exciting terminal emulator since I first found urxvt-unicode!

Second, while the magic chords you've chosen to manipulate windows and panes in Kitty make a lot of sense, I find such multi-key contortions both esoteric to remember and ergonomically "challenging".  

I've been a user of I3WM and TMux w/urxvt for years now.   One thing I love about that config is that I seldom have to remember more than the "meta" key to initiate an action and the keystrokes following a meta key press can be/are mapped to VIM-like motions.  For example:
In VIM, ctrl-w followed by h, j, k, or l moves the focus to the pane left, right, up or down.
In TMUX, ctrl-b followed h, j, k, or l does exactly the same thing.
In I3WM meta + h, j, k, or l also does the same thing (albeit as a keyboard chord).

Is it possible to "trigger" a Kitty command with the prefix of "some key" like cntrl-a followed by a normal ey press (such as h, j, k, and l) to effect the same kind of active window/pane selection?  

How might I map such a thing in Kitty?

**Comment by kovidgoyal** [comment]
kitty support multi-key shortcuts, see
https://sw.kovidgoyal.net/kitty/conf/#keyboard-shortcuts (search for
multi-key)

map ctrl+a>h whatever
